generic/tests/cfg/nfs_corrupt.cfg

```diff
@@ -16,9 +16,7 @@
     post_command_noncritical = yes
     wait_paused_timeout = 240
     nfs_stat_chk_re = "running"
-    s390x:
-        bootindex_image1 = 0
-        bootindex_stg = 1
+    blk_extra_params_stg = "serial=TARGET_DISK"
```

Comment:
suggest change to like :
serial=TARGET_DISK
blk_extra_params_stg += "serial=${serial}"

Reply:
updated